### PR TITLE
[IN-209][Preprints] Add provider description to carousel

### PR DESCRIPTION
**Pull request by fabmiz**
## Purpose

Add provider description to carousel

## Summary of Changes

Add `selectedProvider` description to preprint submit template

## Side Effects / Testing Notes

N/A

## Ticket

[IN-209](https://openscience.atlassian.net/browse/IN-209)

## After
screen shot 2018-04-04 at 3 38 50 pm

screen shot 2018-04-27 at 10 37 19 am

screen shot 2018-04-27 at 10 38 26 am

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: app/templates/submit.hbs [hunk 1]
```diff
@@ -23,7 +23,16 @@
                                 {{preprint-form-header editMode=editMode serverSaveState=serverSaveState showValidationIndicator=true name=name isValidationActive=(and providerSaved (not uploadValid)) currentProvider=currentProvider valid=(not providerChanged)}}
                                 {{#preprint-form-body}}
                                     {{provider-carousel itemsPerSlide=10 providers=allProviders selectable=true activeProvider=selectedProvider selectAction=(action 'selectProvider')}}
-
+                                    {{#if selectedProvider}}
+                                        <div class="row">
+                                            <div style="padding: 25px 85px 15px;" class="col-md-12">
+                                                <dl class="dl-horizontal dl-description">
+                                                    <dt>{{selectedProvider.name}}:</dt>
+                                                    <dd>{{sanitize-html selectedProvider.description "description"}}</dd>
```
Comment (alexschiller) [inline app/templates/submit.hbs]: Arabixiv, for example, has a fair amount of other elements in their description (they have the most complex description that I saw). Can you reply with a screenshot with their description and show how it renders?

```"<p class=\"MsoNormal\"><span> </span><span>الأرشيف العربي </span><span>للنشر</span><span> العلمي على الإنترنت</span><span></span></p>\n<p class=\"MsoNormal\"><span> </span></p>"```

Comment (fabmiz) [reply app/templates/submit.hbs]: With `sanitize-html` help, we strip out all html tags as specified below, leaving only the text. https://github.com/CenterForOpenScience/ember-osf-preprints/blob/6eeb1f957fe5364d1a55b0e8d48e8e34290b21ce/app/sanitizers/description.js#L1-L6.

Comment (fabmiz) [reply app/templates/submit.hbs]: @alexschiller I was wrong here. The above actually means to live out the specified tags (in the elements and attributes list) and strip out every other tag. `sanitize-html selectedProvider.description` strips out all html.

**Comment by alexschiller** [comment]
TCI was green before fixing CHANGELOG, merging.

**Pull request merged**